backend/src/test/java/com/github/astridottmann/backend/services/RouteServiceTest.java

```diff
@@ -19,34 +21,38 @@ class RouteServiceTest {
     RouteService routeService;
     final IdService idService = mock(IdService.class);
     final CalculateCo2EmissionService calculateCo2EmissionService = mock(CalculateCo2EmissionService.class);
-
+    final CompareRoutesService compareRoutesService = mock(CompareRoutesService.class);
+    final CompareRoutesRepository compareRoutesRepository = mock(CompareRoutesRepository.class);
     final RouteRepository routeRepository = mock(RouteRepository.class);
     private final String testId = "1";
     private final double dummyEmission = 162;
 
     private Route createTestRouteInstance() {
+        Car car = new Car("car", 2.8, "petrol", "large");
```

Comment:
I like it with the createTestRouteInstance() methods. Perhaps you can put it in a RouteFactory to use it in each test class. I wrote it in detail in another comment.

Reply:
I will do that1